### ISSUE TITLE
FULLY uninstall docker desktop and any files it's ever created

### ISSUE THREAD

**Issue by rlewkowicz** [docker/for-mac#6345]
I'm trying to downgrade, and I have tried various uninstall scripts including 
```
/Applications/Docker.app/Contents/MacOS/Docker --uninstall
```
and one for docker toolkit

SOMETHING somewhere is telling this that I have k8s enabled and it is just stuck trying to start. I need to wipe EVERYTHING as if it was a fresh install.

**Comment by rlewkowicz** [comment]
Yeah, this is wild. I have tried to find every file on this system that could be docker related. The fact that this isn't well documented, or at least anywhere I could find is crazy.

**Comment by apm963** [comment]
Coincidentally I reached out to support earlier this week with this same question. Here is what they said:

> 1. Uninstall Docker Desktop - Go to the 'Troubleshooting' section (bug icon in the the top right corner of the Docker Desktop Window), and uninstall from here.
> 2. Remove the below files (note that removing these files is destructive, meaning all images, volumes or Docker related files would be deleted, so kindly take note of this point)
> ```sh
> rm -rf ~/Library/Group\ Containers/group.com.docker
> rm -rf ~/Library/Containers/com.docker.docker
> rm -rf ~/.docker
> ```

I also checked with AppCleaner and that provided this list
<img width="582" alt="image" src="https://user-images.githubusercontent.com/3219297/172905304-73955570-e7d6-40f6-bd11-dcad93b9acc7.png">

YMMV.
